ow_lander/scripts/path_planning_commander.py

```diff
@@ -223,17 +219,21 @@ def run(self):
         '/guarded_move_result', GuardedMoveFinalResult, queue_size=10)
     self.guarded_move_srv = rospy.Service(
         'arm/guarded_move', GuardedMove, self.handle_guarded_move)
-    print("path_planning_commander has started!")
+    
+    rospy.loginfo("path_planning_commander has started!")
 
     rospy.spin()
 
-  def return_message(self, action_name, success=True):
-    if not self.arm_fault:
-      print(action_name + " activity completed")
-      return success, "Done"
+  def log_started(self, activity_name):
+    rospy.loginfo("%s arm activity started", activity_name)
+
+  def log_finish_and_return(self, activity_name, success=True):
+    if self.arm_fault:
+      rospy.logerror("%s activity incomplete", activity_name)
+      return False,  "%s failed" % activity_name
     else:
-      print(action_name + " activity incomplete")
-      return False,  action_name + " failed"
+      rospy.loginfo("%s activity completed", activity_name)
+      return success, "Done"
```

Comment:
There is inconsistency in what of some of the finish message printed across different activities. For example, some mention "Guarded Move activity completed" and some would be "Stow arm activity completed". Did you intentionally want this? To me logically you should have just done as follows and remove the word _arm_ when this method is invoked:
```suggestion
      rospy.loginfo("%s arm activity completed", activity_name)
      return success, "Done"
```

Reply:
I agree. This seems more consistent. Thanks.